src/element/annotations.rs

```diff
@@ -17,13 +17,14 @@ pub struct Annotations {
 
 impl Annotations {
     // This is limited to crate visibility to allow us to change `Vec<_>` to something
-    // else (`thinvec`?) in the future.
+    // else (`thinvec`?) in the future. Users are unlikely to need to construct an `Annotations`
+    // themselves, but can use a `From` impl if necessary.
     pub(crate) fn new(symbols: Vec<Symbol>) -> Self {
         Annotations { symbols }
     }
 
-    // Constructs an Annotations object representing an empty symbol sequence
-    pub(crate) fn empty() -> Self {
+    /// Constructs an Annotations object representing an empty symbol sequence
+    pub fn empty() -> Self {
```

Comment:
I made this public for two reasons:
1. It's already possible to construct an `Annotations`, empty or otherwise, using a `From` implementation.
2. I needed an empty `Annotations` for the doc tests and making one via `From` required being annoyingly explicit.

Reply:
3. Constructing an empty `Annotations` does not (cannot?) expose any of the implementation details, so really, the question should be "why not?".